scripts/generate-patch

```diff
@@ -39,8 +39,19 @@ for f in packages/*; do
 				curl -sLf ${url} | tar xvzf - -C ${f}/charts-original --strip ${fields} ${subdirectory} > /dev/null 2>&1
 			fi
 			if [[ -d ${f}/charts ]]; then
+				if [[ -d ${f}/charts-crd ]]; then
```

Comment:
Would be good to add comment here explaining why you need to move crds around when patching.

Reply:
Will be incorporated in the next commit